CONTRIBUTING.md

```diff
@@ -8,6 +8,16 @@ While this project is stewarded by [WordPress AI Team](https://make.wordpress.or
 
 All parameters, return values, and properties must use explicit type hints, except in cases where providing the correct type hint would be impossible given limitations of the oldest supported PHP version (see below).
 
+## Naming conventions
+
+The following naming conventions must be followed for consistency and autoloading:
+
+- Interfaces are suffixed with `Interface`
+- Traits are suffixed with `Trait`
+- Enums are suffixed with `Enum`
+- File names are the same as the class, trait, and interface name for PSR-4 autoloading
+- Classes, interfaces, and traits, and namespaces must not be prefixed with `Ai`, exluding the root namespace.
```

Comment:
Nit-picks: Let's consistently use punctuation :)

Also I'm suggesting to adjust the wording in the last one to match how the other ones are worded:
```suggestion
- Interfaces are suffixed with `Interface`.
- Traits are suffixed with `Trait`.
- Enums are suffixed with `Enum`.
- File names are the same as the class, trait, and interface name for PSR-4 autoloading.
- Classes, interfaces, and traits, and namespaces are not prefixed with `Ai`, exluding the root namespace.
```